integrations/axum/src/lib.rs

```diff
@@ -1275,6 +1278,7 @@ pub async fn build_static_routes<IV>(
         .await
         .expect("could not build static routes")
     });
+    let _ = handle.await;
```

Comment:
`leptos_reactive::spawn_local` (like the underlying [`wasm_bindgen_futures::spawn_local`](https://docs.rs/wasm-bindgen-futures/latest/wasm_bindgen_futures/fn.spawn_local.html)) returns `()`, not a handle that can be awaited, so you can see the following error in CI when the `wasm` feature is enabled:
```
 error[E0277]: `()` is not a future
    --> integrations/axum/src/lib.rs:1281:20
     |
1281 |     let _ = handle.await;
     |                   -^^^^^
     |                   ||
     |                   |`()` is not a future
     |                   help: remove the `.await`
     |
     = help: the trait `futures::Future` is not implemented for `()`, which is required by `(): std::future::IntoFuture`
     = note: () must be a future or must implement `IntoFuture` to be awaited
     = note: required for `()` to implement `std::future::IntoFuture`
```

Reply:
Hadn't seen that CI was finally running 🥳 

Hmm, while I've got rid of all the errors, I did end up with a bit of an iffy solution to it, essentially:

```
    #[allow(unused_variables, clippy::let_unit_value)]
    let handle = spawn_task!(async move {
        leptos_router::build_static_routes(
            &options,
            app_fn,
            &routes,
            &static_data_map,
        )
        .await
        .expect("could not build static routes")
    });
    #[cfg(all(not(feature = "wasm"), feature = "default"))]
    let _ = handle.await;
```

I'd be curious if you have any suggestions to alternatives? For now I've been trying to be minimally invasive, but that might be the wrong approach here 🤔